src/editor/Core/Reducer.re

```diff
@@ -51,8 +51,7 @@ let updateTabs = (bufId, modified, tabs) =>
     |> sortTabsById
   );
 
-let applyBufferUpdate =
-    (bufferUpdate: BufferUpdate.t, buffer: option(Buffer.t)) =>
+let applyBufferUpdate = (bufferUpdate, buffer) =>
```

Comment:
these are type inferred correctly, I'm never sure whether to annotate for readability? but given how good ocaml is at inference wasn't sure if we should save annotations for when the compiler needs the extra help?

Reply:
I'm ok removing types if they are not needed, inferred! 👍